agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/localstorage/LocalFileWriter.java

```diff
@@ -123,8 +123,7 @@ private static long getTotalSizeOfPersistedFiles(File telemetryFolder) {
     }
 
     long sum = 0;
-    Collection<File> files =
-        FileUtils.listFiles(telemetryFolder, new String[] {PERMANENT_FILE_EXTENSION}, false);
+    Collection<File> files = FileUtils.listFiles(telemetryFolder, new String[] {"trn"}, false);
```

Comment:
oh, were we writing/reading `*..trn` files before? 😁

Reply:
permanent file name is .trn.. but when we use FileUtils.listFiles, it has to be the file extension without dot.  That's the bug. with ".trn", listfiles always return nothing.

Reply:
do we have any tests that would/should catch this issue?

Reply:
https://github.com/microsoft/ApplicationInsights-Java/pull/1860/files#:~:text=testDeleteFilePermanentlyOnFailure

i found this bug when writing this test. what other test do you have in mind? just test listFiles?  It's from Commons-Utils.

Reply:
```
Finds files within a given directory (and optionally its subdirectories) which match an array of extensions.
Params:
directory – the directory to search in
**extensions – an array of extensions, ex. {"java","xml"}. If this parameter is null, all files are returned.**
recursive – if true all subdirectories are searched as well
Returns:
a collection of java.io.File with the matching files
```

java doc for FileUtils.listFiles is pretty clear.

Reply:
great, so the new test you added would have caught this?